template/page-community.html.jinja

```diff
@@ -17,386 +17,228 @@
+              <p class="project-description">{% trans %}Your go-to independent blockchain infrastructure provider, offering secure and scalable blockchain services, including wallets, nodes, staking, protocols, and integrations for developers and institutions alike.{% endtrans %}</p>
+              <a href="https://xrpl.org/blog/2023/blockdaemon.html" target="_blank" class="view-project external-link">{% trans %}View Project{% endtrans %}</a>
+            </div>
         </div>
-        <div class="mb-4 pb-3 mb-lg-3 pb-lg-5">
-          <img alt="Blue and pink card" src="./assets/img/community/community-events-hackathon@2x.png" class="w-100">
-          <h6 class="mt-3">{% trans %}XRPL Hackathon 2022{% endtrans %}</h6>
+        <div class="project-card bottom-right">
+              <div class="card-image">
+              <img class="middle-image-two" src="./assets/img/community/xrp-cafe.png" />
+              </div>
+            <div class="card-details">
+              <h6 class="project-title">XRPCafe</h6>
+              <p class="project-description">{% trans %}A premier NFT marketplace dedicated to fostering mass adoption of the XRP Ledger.{% endtrans %}</p>
+              <a href="https://xrpl.org/blog/2023/xrpcafe.html" target="_blank" class="view-project external-link">{% trans %}View Project{% endtrans %}</a>
+            </div>
         </div>
     </div>
-      
-      <div >
-        <a class="btn btn-primary btn-arrow" target="_blank" href="/events.html">{% trans %}View All Events{% endtrans %}</a> 
-      </div>
-    </div>
-    
-  </div>
 </section>
 
-
-<!-- Careers -->
-<section class="container-new" id="xrpl-careers">
-  <div class="card-grid card-grid-2xN">
-    
-    <div class="col pr-2 d-lg-block d-none">
-      <div class="mb-4 pb-3" id="careersDark"></div>
-    </div>
-    
-    <div class="col pt-5">
-      <div class="d-flex flex-column-reverse mb-lg-2 pl-0">
-        <h2 class="h4 h2-sm">{% trans %}Discover your next career opportunity in the XRPL community{% endtrans %}</h2>
-        <h6 class="eyebrow mb-3">{% trans %}XRPL Careers{% endtrans %}</h6>
-      </div>
-      <p class="mb-3 py-4">{% trans %}Teams across the XRPL community are looking for talented individuals to help build their next innovation.{% endtrans %}</p>
-      
-      <div class="d-lg-none d-block">
-        <div class="mb-4 pb-3" id="careersDark-small"></div>
+<section class="bottom-cards-section">
+  <div class="com-card">
+    <img class="top-left-img" />
+    <div class="card-content">
+      <h6 class="card-title">{% trans %}Contribute to Consensus{% endtrans %} </h6>
+      <h6 class="card-subtitle">{% trans %}Run an XRP Ledger network node{% endtrans %} </h6>
+      <p class="card-description">{% trans %}Thank you for your interest in contributing to XRPL.org. {% endtrans %}</p>
+      <div class="card-links">
+        <a class="com-card-link" target="_blank" href="the-rippled-server.html">{% trans %}The Rippled Server{% endtrans %}</a>
```

Comment:
Two things:

1. This link text fails WCAG contrast guidelines. With foreground #7919FF, background #232325, it has contrast ratio 2.53:1. The guideline AA for "normal text" (this isn't large text) is 4.5:1.
2. The `rippled` server name is written in all lowercase, preferably code font.
```suggestion
        <a class="com-card-link" target="_blank" href="the-rippled-server.html">{% trans %}The <code>rippled</code> Server{% endtrans %}</a>
```